assets/src/apps/delivery/store/features/adaptivity/actions/triggerCheck.ts

```diff
@@ -91,7 +90,22 @@ export const triggerCheck = createAsyncThunk(
         }
       });
     });
-    const stateSnapshot = { ...allResponseState, ...modifiedExtrinsicState };
+
+    const spanshot = getEnvState(defaultGlobalEnv);
+    const getGlobalSnapshot = [spanshot].reduce((collect: any, variableSnappshot: any) => {
```

Comment:
you don't need to wrap the object into an array for this reduce, just start with
`const globalSnapshot = Object.keys(snapshot).reduce((collect: any, key: string) => {
  if (key.indexOf('app.') === 0 || key.indexOf('variables.') === 0) {
          collect[key] = snapshot[key];
        }
return collect;
});
```